pkg/ccl/backupccl/restore_job.go

```diff
@@ -281,9 +343,17 @@ func restore(
 	// which are grouped by keyrange.
 	highWaterMark := job.Progress().Details.(*jobspb.Progress_Restore).Restore.HighWater
 
-	importSpans := makeSimpleImportSpans(dataToRestore.getSpans(), backupManifests,
+	var importSpans []execinfrapb.RestoreSpanEntry
+	importSpans = makeSimpleImportSpans(dataToRestore.getSpans(), backupManifests,
```

Comment:
nit: delete line 346 and create importSpans in one line:

```
importSpans := makeSimpleImportSpans(...)
```

Reply:
also, could you add a TODO within makeSimpleImportSpans to remove the `highWaterMark` argument in 23.1?